apps/vmq_server/src/vmq_cluster_node.erl

```diff
@@ -201,20 +205,32 @@ handle_message({connect_async_done, AsyncPid, {ok, {Transport, Socket}}},
     Msg = [<<"vmq-connect">>, <<L:32, NodeName/binary>>],
     case send(Transport, Socket, Msg) of
         ok ->
-            lager:info("successfully connected to cluster node ~p", [RemoteNode]),
-            State#state{socket=Socket, transport=Transport,
+            lager:info("cluster_node ~p : successfully connected to cluster node ~p", [self(), RemoteNode]),
+            NewState = State#state{socket=Socket, transport=Transport,
                         %% !!! remote node is reachable
                         async_connect_pid=undefined,
-                        reachable=true};
+                        reachable=true,
+                        waiting_for_ack=false},
+            internal_flush(NewState);
         {error, Reason} ->
-            lager:warning("can't initiate connect to cluster node ~p due to ~p", [RemoteNode, Reason]),
+            lager:warning("cluster_node ~p : can't initiate connect to cluster node ~p due to ~p", [self(), RemoteNode, Reason]),
```

Comment:
Should we use `self()` (the pid) here or `node()`?

Reply:
I agree, node() is better, will adapt it.

Reply:
Thanks 🎉